src/components/IonImageSettings.vue

```diff
@@ -38,7 +38,7 @@
    },
    data() {
      return {
-       availableScales: ["Viridis", "Hot", "Greys", "Portland", "YlGnBu"]
+       availableScales: ["Cividis","Viridis", "Hot", "Greys", "Portland", "YlGnBu"]
```

Comment:
@Ren22 can you order them ["Viridis", "Cividis", "Hot",  "YlGnBu", "Portland", "Greys"]

Reply:
yes, will change it @andy-d-palmer